core/src/test/scala/integration/kafka/tools/DumpLogSegmentsIntegrationTest.scala

```diff
@@ -0,0 +1,66 @@
+package integration.kafka.tools
```

Comment:
Add license header.
Otherwise Jenkins wouldn't run the test.

Reply:
Just added

Reply:
The package should be `kafka.tools` to be consistent with the other test cases.